src/components/calculator/Calculator.jsx

```diff
@@ -0,0 +1,130 @@
+import React, { useState } from "react";
+import Display from "./Display";
+import Button from "./Button";
+
+function Calculator() {
+    const [number, setNumber] = useState(0);
+    const [oldNumber, setOldNumber] = useState(0);
+    const [operator, setOperator] = useState();
```

Comment:
It's not relevant but to make it easier to understando, specify that the initial value of the `operator` is null

Reply:
It already does that because it's JS, but it makes more sense to specifiy that it starts with null or just an empty string like ""

Reply:
empty string added.